news/2023/2023-05-01-journey-into-a-beatmap-world-contest-results.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: post
+title: "Journey into a Beatmap World Art Contest: Results"
+date: 2023-05-01 07:00:00 +0000
+---
+
+We've travelled across the (beatmapping) world to get you these results, now showing on a main menu near you!
+
+![](https://assets.ppy.sh/contests/175/banners/w.jpg)
+
+2944 intrepid explorers joined us in this great trek across the expanded osu! universe, from [ostenatious openings](https://assets.ppy.sh/contests/175/entries/Trembling%20Cheetah.jpg), [programs most strange](https://assets.ppy.sh/contests/175/entries/Jolly%20Chipmunk.jpg) and even [disjointed backroom photo shoots](https://assets.ppy.sh/contests/175/entries/Stern%20Magpie.jpg), we've seen it all.
+
+But the architects of this terrible journey decreed that you shall only have fifteen votes to spend across such a huge collection of insanely talented works. And so, you have chosen.
+
+Let us witness where your travels have taken us:
+
+[![](https://assets.ppy.sh/contests/175/banners/w.jpg)](https://assets.ppy.sh/contests/175/winners/w.png)
+
+**1312** voters descended into the neverending torment of [**\]w\[**](https://osu.ppy.sh/users/16964067)'s most excellent rendition of the infamous [In Putris Stagnum](https://osu.ppy.sh/beatmapsets/968625#osu/2026764) planescape! pippi looks on, bathed in the light of the abyss.
+
+And the abyss stares back.
+
+Congratulations to \]w\[, who takes their first top spot on their third overall entry into one of our art contests! Well-earned and super deserved, playing to the contest's theme masterfully.
+
+[![](https://assets.ppy.sh/contests/175/banners/rxys.jpg)](https://assets.ppy.sh/contests/175/winners/rxys.png)
+
+A fool moon night rises over **1201** voters who did not walk away from [**rxys**](https://osu.ppy.sh/users/22185836)'s rendition of [one of osu!'s most iconic beatmaps](https://osu.ppy.sh/beatmapsets/524026#osu/1186901), joined by a most monochrome pippi and a score-surfing WhiteCat.
+
+Some say that if you look hard enough, you can see two visitors from beyond among the stars...
```

Comment:
this supposed second one is eluding me, you sure you didn't miscount?

Reply:
(the amogus and The Number)

Reply:
also this should be either visitors from beyond OR among the stars, good accidental catch

Reply:
it's not actually an error. you can put a comma for pause to avoid the garden path

```suggestion
Some say that if you look hard enough, you can see two visitors from beyond, among the stars...
```

i kinda disagree that "the number" counts as a "visitor" though. it doesn't seem like you're referring to both easter eggs here with the strong word-choice-assisted hinting